app/javascript/components/workers-form/workers-form.jsx

```diff
@@ -20,57 +20,55 @@ const WorkersForm = ({ server: { id, name }, product, zone }) => {
       .then(({ workers: { worker_base } }) => {
         const wb = worker_base;
 
-        const countDefault = wb.defaults.count;
-        const memDefault = toBytes(wb.defaults.memory_threshold);
-        const baseMemDefault = toBytes(wb.queue_worker_base.defaults.memory_threshold);
-        const monitorDefault = toBytes(wb.event_catcher.defaults.memory_threshold);
-
-        const selectCount = value => (typeof value === 'number' ? value : countDefault);
```

Comment:
Only thing I'm curious about is what this `=== 'number'` means... @kavyanekkalapu do you know?

Reply:
Maybe it's to handle `0` which is a number but is also falsey?

Reply:
Note, we also don't check for countDefault to be a number but theoretically, that could be empty/"".  We might need validation on all worker values + all `*Default` values here.  I can't tell.